src/sagas/api.js

```diff
@@ -4,74 +4,115 @@ import { call, select } from 'redux-saga/effects';
 import { getToken } from '../selectors/account';
 import { getApiRoot, getTokenType } from '../selectors/env';
 
-export function* getHeaders (authenticationRequired) {
+import * as _ from 'lodash';
+
+const parseOptions = (options) => {
+  let data = null;
+  let customHeaders = undefined;
+
+  if (_.has(options, 'data')) {
+    data = _.get(options, 'data');
+
+    if (_.has(options, 'headers')) {
```

Comment:
Do we only want this if there is data?  What if we want custom headers on a `get` request that doesn't have data?

Reply:
+1